sig-cluster-lifecycle/annual-report-2021.md

```diff
@@ -0,0 +1,99 @@
+    * [Recent mailing list notification](https://groups.google.com/g/kubernetes-sig-cluster-lifecycle/c/KH8g4WRjOAE)
+    * [https://github.com/kubernetes/kubernetes/pull/98547](https://github.com/kubernetes/kubernetes/pull/98547)
+    * [https://github.com/kubernetes/kubeadm/pull/2390](https://github.com/kubernetes/kubeadm/pull/2390)
+* Same question as above but for working groups.
+  * The [WG Component Standard ](https://git.k8s.io/community/wg-component-standard) which we co-own with
+  SIG API Machinery [is looking for new leads](https://groups.google.com/g/kubernetes-wg-component-standard/c/sNAqDptjJug/m/P87KtaZuAgAJ).
+  On private we proposed that we should ideally find new leads instead of disbanding the WG,
+  but if no leads volunteer we are going to have to do that. The WG is not very active, due to the current
+  leads being busy with other work.
+  * We do not own other WGs.
+* When was your last monthly community-wide update? (provide link to deck and/or recording)
+  * Last presentation was on July 16, 2020:
+    * [Slides](https://docs.google.com/presentation/d/1ZEDeF6lqxP-LmxCRa2EBmDS1sZFAv3RmrdQOUyd6IAc/edit?usp=sharing)
+    * [VOD](https://www.youtube.com/watch?v=J3O8fXTm3HE)
+
+## Membership
+
+* Are all listed SIG leaders (chairs, tech leads, and subproject owners) active?
+  * For the SIG leads yes.
+  * For subproject owners, likely there are some inactive maintainers, but it’s up to the subproject to prune their OWNERs files.
+* How do you measure membership? By mailing list members, OWNERs, or something else?
+  * Anyone can be considered a SIG member if they join the Zoom calls or general discussions regularly.
+  * In terms of repository scope - an active contributor with PRs / Issues / Reviews becomes eligible to be part of an OWNERS file.
+* How does the group measure reviewer and approver bandwidth? Do you need help in any area now? What are you doing about it?
+  * For sub-projects, it’s up to the active subproject maintainers to allocate review resources.
+  * Some projects like etcdadm, cluster-addons and kubeadm are currently looking for more contributors.
+  * We use the common methods of
+  [“help-wanted” labels](https://github.com/kubernetes-sigs/cluster-api/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22)
+  and announcing the request for help on slack / mailing list.
+  * For the main SIG (e.g. when reviewing KEPs) the lead who has the time usually reviews, but we try to notify everyone
+  so that they can comment if they have the time too.
+* Is there a healthy onboarding and growth path for contributors in your SIG?
+What are some activities that the group does to encourage this? What programs are you participating in to grow contributors
+throughout the contributor ladder?
+  * One of our methods is to just record onboarding videos / hold Zoom sessions discussing a certain problem / area:
```

Comment:
What about adding that most of the meetings have a dedicated session to welcome newcomers?